cmd/influxd/inspect/verify_seriesfile_test.go

```diff
@@ -44,25 +51,21 @@ func TestVerifies_Invalid(t *testing.T) {
 		defer test.Restore(path)
 
 		fh, err := os.OpenFile(path, os.O_RDWR, 0)
-		test.AssertNoError(err)
+		require.NoError(t, err)
 		defer fh.Close()
 
-		_, err = fh.WriteAt([]byte("BOGUS"), 0)
-		test.AssertNoError(err)
-		test.AssertNoError(fh.Close())
+		_, err = fh.WriteAt([]byte("foobar"), 0)
+		require.NoError(t, err)
+		require.NoError(t, fh.Close())
 
-		passed, err := seriesfile.NewVerify().VerifySeriesFile(test.Path)
-		test.AssertNoError(err)
-		test.Assert(!passed)
+		passed, err := NewVerify().VerifySeriesFile(test.Path)
```

Comment:
Should we set a zaptest logger on this one as well, to avoid noise when the test succeeds?